libcodechecker/server/client_auth_handler.py

```diff
@@ -75,5 +68,5 @@ def performLogin(self, auth_method, auth_string):
 
     @timeit
     def destroySession(self):
-        return self.__manager.invalidate(self.__client_host,
+        return SessionManager.invalidate(self.__client_host,
```

Comment:
Same comment as the changes in `libcodechecker/instance_manager.py`. The static method here breaks symmetry. What if a module overrides `self.__manager` in this class?